examples/rust_control_interface/src/main.rs

```diff
@@ -27,16 +32,32 @@ use std::{
     fs::File,
     io,
     io::{Read, Write},
-    path::Path,
+    path::{Path, PathBuf},
     process::exit,
     time::Duration,
 };
 
+// =======================================================================
+// Variables - general
+// =======================================================================
+// - ANKAIOS_CONTROL_INTERFACE_BASE_PATH is the path to the input and output
+//     FIFO files of the control interface.
+// - MAX_VARINT_SIZE is the maximum size of a varint encoded data.
+// - WAITING_TIME_IN_SEC represents the default waiting time.
+// - UPDATE_STATE_REQUEST_ID is the request ID for the update state request.
+// - COMPLETE_STATE_REQUEST_ID is the request ID for the complete state request.
+// - PROTOCOL_VERSION is the version of Ankaios, which for this example
+//     is set to the value of the environment variable 'ANKAIOS_VERSION'.
 const ANKAIOS_CONTROL_INTERFACE_BASE_PATH: &str = "/run/ankaios/control_interface";
 const MAX_VARINT_SIZE: usize = 19;
 const WAITING_TIME_IN_SEC: u64 = 5;
-const REQUEST_ID: &str = "dynamic_nginx@rust_control_interface";
+const UPDATE_STATE_REQUEST_ID: &str = "dynamic_nginx@12345";
+const COMPLETE_STATE_REQUEST_ID: &str = "dynamic_nginx@67890";
```

Comment:
Let's keep it simple:
```suggestion
const UPDATE_STATE_REQUEST_ID: &str = "12345";
const COMPLETE_STATE_REQUEST_ID: &str = "67890";
```